### ISSUE TITLE
I can't find the pre trained model of LEDNet

### ISSUE THREAD

**Issue by jaggernaut007** [AceCoooool/LEDNet#2]
If the model cannot be provided. May I know how long will the training take to converge?

**Comment by AceCoooool** [comment]
I did not find the training epoch and whether pretrained model in imagenet in the paper. (And also not sure the init strategy used in paper)
There are two experiments I have done (using pytorch default init)
- Base Size 1024, Crop Size 768, only fine.

| Model  | Paper | OHEM | Epoch | val (crop)  |     val     |
| :----: | :---: | :--: | :---: | :---------: | :---------: |
| LEDNet |   /   |  ✗   |  240  | 44.67/91.85 | 49.79/91.31 |
| LEDNet |   /   |  ✗   | 1000  | 53.77/93.45 | 59.04/93.27 |

**Comment by xiaoyufenfei** [comment]
@AceCoooool   Hi, Cityscapes size is 1024x512 during training. At the same time, Dropout should be set for each branch of SS-nbt building block in basic.py. LEDNet is inspired by ENet. You can refer to the Dropout setting in ENet. For example, if you set a small drop_rate in the shallow layer and a large drop_rate in the middle 8th layer, I am sorry that the paper does not describe it clearly. In addition, all the bias should be set to True when training from the scratch. I hope this helps you.

**Comment by jaggernaut007** [comment]
So led net is similar to enter, apart from the dropouts? I am trying the
eespnet as I couldn't get a working model architecture running from my side
on pytorch.

**Comment by AceCoooool** [comment]
@xiaoyufenfei In my experiment, some setting is different with your paper:
1. the learning rate is too small in your paper (5e-4), I find it's better to use larger learning rate (like 0.00125*batch-size as most segmentation paper use---although these papers use pretrained model)
2. the value of drop-rate in dropout seems not influence the performance much.

Thank you for your kind help.

**Comment by AceCoooool** [comment]
@jaggernaut007 I have provided the [pretrained model](https://drive.google.com/open?id=1oelPUKAnZYD75RruyBQU9HZKneMEMIAp). (It's nearly close to the original paper)
Note: I use different input-size as training setting

**Comment by xiaoyufenfei** [comment]
I really admire your work. For some reasons, I have not been able to upload the code in time. I hope that after uploading, it can improve your results. Indeed, it is necessary to perform pre-training and fine-tuning the result will better. good job.

**Comment by jaggernaut007** [comment]
Thanks for your support. Nice job!
